extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -273,18 +276,15 @@ public function getWikiVariables() {
 
 		$wikiVariables = $this->mercuryApi->getWikiVariables();
 
-		try {
-			$wikiVariables['navData'] = $this->getNavigationData();
-		} catch ( Exception $e ) {
+		$navData = $this->getNavigationData();
+		if ( empty( $navData ) ) {
 			\Wikia\Logger\WikiaLogger::instance()->error(
```

Comment:
I wanted to leave this error here because there was a lot of errors reported in Kibana and I hope this should fix all of them but if not it's good to have some message somewhere that it's still not working